test_scripts/Defects/8_0/3541_Resumption_of_MenuLayout.lua

```diff
@@ -0,0 +1,143 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/3541
+---------------------------------------------------------------------------------------------------
+-- Steps:
+-- 1. App sets menu layout through 'SetGlobalProperties'
+-- 2. App adds sub menu with menu layout through 'AddSubMenu'
+-- 3. App unexpectedly disconnects and reconnects with correct 'hashId'
+--
+-- SDL does:
+--  - start data resumption process and send 'UI.SetGlobalProperties' and 'UI.AddSubMenu' requests to HMI
+--  - both requests contains 'menuLayout' parameter
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+local utils = require("user_modules/utils")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local hashId
+local sgpParams = {
+  menuTitle = "menuTitle",
+  menuLayout = "TILES"
+}
+local asmParams = {
+  menuLayout = "TILES",
+  menuID = 44991234,
+  menuName = "menuName"
+}
+
+--[[ Local Functions ]]
+local function sendWindowCapabilities()
+  local onSysCaps = {
+    systemCapability = {
+      systemCapabilityType = "DISPLAYS",
+      displayCapabilities = {
+        {
+          windowCapabilities = {
+            {
+              menuLayoutsAvailable = { "LIST", "TILES" }
+            }
+          }
+        }
+      }
+    }
+  }
+  onSysCaps.appID = common.getHMIAppId()
+  common.getHMIConnection():SendNotification("BasicCommunication.OnSystemCapabilityUpdated", onSysCaps)
+end
+
+local function sendSetGP()
+  common.getMobileSession():ExpectNotification("OnHashChange")
```

Comment:
@dboltovskyi I propose to avoid using old functions of actions module like  `actions.getMobileSession()` because it make harder future refactoring of scripts. Please use `actions.mobile.getSession()` and `actions.mobile.createSession()` instead of `actions.getMobileSession()`

Reply:
Please find update in 167e57a